config/locales/landing_pages.yml

```diff
@@ -232,11 +232,6 @@ en:
       meta_description: "Search for Spanish teacher jobs near you on Teaching Vacancies. Find the latest full time and part time jobs for spanish teachers in schools across England."
 
     ### Working patterns
-    flexible-working-jobs-in-schools:
```

Comment:
This removes these links from the homepage:
![Screenshot 2023-07-24 at 15 44 31](https://github.com/DFE-Digital/teaching-vacancies/assets/13124899/2786cf83-f626-4402-b529-9e0942cb90b8)

Reply:
Thanks for the screenshot! I was puzzled!